External/FEXCore/Source/Interface/Core/Core.cpp

```diff
@@ -519,8 +519,14 @@ namespace FEXCore::Context {
     State->PassManager->RegisterExitHandler([this]() {
         Stop(false /* Ignore current thread */);
     });
-
-    State->PassManager->AddDefaultPasses(Config.Core == FEXCore::Config::CONFIG_IRJIT);
+    
+    #if _M_ARM_64
```

Comment:
Config option enablement instead?

Reply:
Follow up PR for configurable SRA regs + this an option